backend/products/views.py

```diff
@@ -17,3 +24,38 @@ def get_queryset(self):
         if category:
             queryset = queryset.filter(category__slug=category)
         return queryset
+
+
+class ProductReviewListCreateView(generics.ListCreateAPIView):
+    serializer_class = ProductReviewSerializer
+
+    def get_queryset(self):
+        product_slug = self.kwargs["product_slug"]
+        return ProductReview.objects.filter(product__slug=product_slug)
+
+    def get_permissions(self):
+        if self.request.method == "POST":
+            return [IsCustomer()]
+        return [permissions.AllowAny()]
+
+    def perform_create(self, serializer):
+        product_slug = self.kwargs["product_slug"]
+        product = Product.objects.get(slug=product_slug)
```

Comment:
If product not found → 500 error,
Better to raise 404 error instead.
Here are my suggested changes:
```suggestion
        from django.shortcuts import get_object_or_404
        product = get_object_or_404(Product, slug=product_slug)
```